core/src/main/scala/kafka/log/stream/s3/telemetry/TelemetryManager.java

```diff
@@ -266,6 +271,37 @@ private SdkMeterProvider getMetricsProvider(Resource resource) {
         return sdkMeterProviderBuilder.build();
     }
 
+    private void initS3Exporter(SdkMeterProviderBuilder sdkMeterProviderBuilder, KafkaConfig kafkaConfig) {
```

Comment:
TODO: auto init s3 metrics exporter when ops bucket specified

Reply:
I will fix it in next pr.